<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecipeWidget</class>
 <widget class="QWidget" name="RecipeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>503</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Recipe</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resources.qrc">
    <normaloff>:/ThreeBrooks/Recipe/recipe.svg</normaloff>:/ThreeBrooks/Recipe/recipe.svg</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,15">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="grpParameters">
       <property name="maximumSize">
        <size>
         <width>240</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Recipe Parameters</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <property name="horizontalSpacing">
         <number>3</number>
        </property>
        <property name="verticalSpacing">
         <number>3</number>
        </property>
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="lblName">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="txtName">
          <property name="toolTip">
           <string>The name of this recipe</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lblStyle">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Style</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmbStyle">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>The style to compare this recipe with</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lblVolume">
          <property name="text">
           <string>Volume</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="spnVolume">
          <property name="toolTip">
           <string>The volume of wort going left after the boil</string>
          </property>
          <property name="suffix">
           <string> gallons</string>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>5.500000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="lblBoilTime">
          <property name="text">
           <string>Boil Time</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="spnBoilTime">
          <property name="toolTip">
           <string>The longest boil time for hops</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="suffix">
           <string> minutes</string>
          </property>
          <property name="decimals">
           <number>0</number>
          </property>
          <property name="maximum">
           <double>1000.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>5.000000000000000</double>
          </property>
          <property name="value">
           <double>60.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="lblEfficiency">
          <property name="text">
           <string>Efficiency</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="spnEfficiency">
          <property name="toolTip">
           <string>Your mash extraction efficiency</string>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>75.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="grpCalculated">
       <property name="title">
        <string>Calculated Values</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTableView" name="tblCalculated">
          <property name="minimumSize">
           <size>
            <width>360</width>
            <height>140</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::DotLine</enum>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="grpIngredients">
     <property name="title">
      <string>Ingredients</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QTreeView" name="trvIngredients">
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="btnRaise">
          <property name="toolTip">
           <string>Move selected ingredients up [Ctrl+Shift+Up]</string>
          </property>
          <property name="text">
           <string>Move Up</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources.qrc">
            <normaloff>:/ThreeBrooks/Recipe/up.svg</normaloff>:/ThreeBrooks/Recipe/up.svg</iconset>
          </property>
          <property name="shortcut">
           <string>Ctrl+Shift+Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnLower">
          <property name="toolTip">
           <string>Move selected ingredients down [Ctrl+Shift+Down]</string>
          </property>
          <property name="text">
           <string>Move down</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources.qrc">
            <normaloff>:/ThreeBrooks/Recipe/down.svg</normaloff>:/ThreeBrooks/Recipe/down.svg</iconset>
          </property>
          <property name="shortcut">
           <string>Ctrl+Shift+Down</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnRemove">
          <property name="toolTip">
           <string>Remove selected ingredients [Del]</string>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources.qrc">
            <normaloff>:/ThreeBrooks/Recipe/remove.svg</normaloff>:/ThreeBrooks/Recipe/remove.svg</iconset>
          </property>
          <property name="shortcut">
           <string>Del</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
